### PR TITLE
Release v1.19.0

### PR DESCRIPTION
**Pull request by StanWarz**
Also applied some files formatting.

### CODE REVIEW

**Review by rchapel — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
 # Changelog
 
-## v.1.19.0
+## 1.19.0
 
-### 09/12/2020
+### 12/10/2020
```
Comment (roxanemace) [inline CHANGELOG.md:5]: Why did you change the date format and the version format?

Comment (StanWarz) [reply CHANGELOG.md:5]: Indeed, because vx.y.z is only the git tag of the x.y.z version.
Also, dates are commonly in US format in changelogs and are not displayed in Github drafts.

Comment (roxanemace) [reply CHANGELOG.md:5]: If you want for the version, but I don't see why we should display dates in US format (we're in Europe)

Comment (StanWarz) [reply CHANGELOG.md:5]: We actually should use the YYYY-MM-DD format according to the norm [ISO-8601](https://en.wikipedia.org/wiki/ISO_8601) for IT aims, do you agree?

Comment (roxanemace) [reply CHANGELOG.md:5]: I would prefer the `YYYY-MM-DD` format, yes.

Comment (StanWarz) [reply CHANGELOG.md]: Applied the right date format.

File: .circleci/config.yml [hunk 2]
```diff
@@ -1,56 +1,47 @@
...
 # Check https://circleci.com/docs/2.0/language-javascript/ for more details
-#
+
 version: 2
 
 defaults: &defaults
   working_directory: ~/repo
   docker:
-    - image: circleci/node:12
+    - image: circleci/node:15
```
Comment (roxanemace) [inline .circleci/config.yml:10]: The change to the CI has nothing to do with the release :)

**Review by roxanemace — APPROVED** [review]

File: .circleci/config.yml (hunk 2 above)
Comment (StanWarz) [reply .circleci/config.yml:10]: But it worked out, I tried with this PR as a draft first :)
